src/main/java/ee/tuleva/onboarding/user/UserService.java

```diff
@@ -44,7 +44,7 @@ public User updateUser(String personalCode, String email, String phoneNumber) {
         findByPersonalCode(personalCode)
             .map(
                 existingUser -> {
-                  existingUser.setEmail(email);
+                  existingUser.setEmail(email.orElse(null));
```

Comment:
Any risk of overwriting existing emails in the database with nulls?

Reply:
Nein. We're calling this from two places: 
1. front-end where the e-mail presence is validated
2. `updateContactDetails` where we're only running the code if both e-mail and phone number are empty so nothing to lose